packages/tables/frontend/sirius-components-tables/src/table/TableContent.tsx

```diff
@@ -200,7 +211,18 @@ export const TableContent = memo(
         },
       },
       renderRowActions: ({ row }) => (
-        <>
+        <div
```

Comment:
This should probably end up being extracted in a dedicated component, it should be soon anyway